### DISCUSSION TITLE
关于本站 – ChloroBlog

### DISCUSSION THREAD

**Discussion by Chlorie** [Chlorie/ChloroBlog#28] [Announcements]
https://chlorie.github.io/ChloroBlog/about/ 

 能鸽善鹉者的领域

**Comment by Constiff3251** [comment]
Hello can you give me the download link for Deenote pls? I downloaded it from Github but can't find an exe file in the zip file and someone told me I downloaded the wrong thing and I have the source code and not the program

**Comment by Chlorie** [comment]
> Hello can you give me the download link for Deenote pls? I downloaded it from Github but can't find an exe file in the zip file and someone told me I downloaded the wrong thing and I have the source code and not the program

https://github.com/Chlorie/Deenote/releases
There's this conveniently placed big "Releases" button on the main page, yet everyone keeps ignoring it and somehow clone the project source code directly. Well done, users.

**Comment by AprilHe** [comment]
哈喽，我很感兴趣你的alphagen repo，但在使用过程中遇到一些问题，方便加个联系方式交流一下吗？谢谢
